medicines/web/src/components/cookie-form/index.tsx

```diff
@@ -57,6 +57,8 @@ const StyledContentWrapper = styled.main`
   }
 `;
 
+const cookieDomain = process.env.ROOT_URL_DOMAIN as string;
```

Comment:
I don't suppose there's any way to default this to `localhost` so it doesn't need to be set when running locally?